app/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.2'
+        classpath 'com.android.tools.build:gradle:2.1.3'
```

Comment:
Please revert this back to 2.1.2. We've had trouble with updating gradle to the latest and greatest due to some of the library dependencies we have. E.g. tests or libraries can fail. If I remember correctly we had to stay with gradle 2.1.2 in order for tests to pass on travis.